ec_grasp_planner/src/planner.py

```diff
@@ -259,11 +294,15 @@ def create_surface_grasp(object_frame, support_surface_frame, handarm_params, ob
     # control_sequence.append(ha.FramePoseSwitch('Pre_preGrasp', 'Pregrasp', controller='GoAboveIFCO', epsilon='0.01'))
 
     # 2. Go above the object - Pregrasp
-    control_sequence.append(ha.InterpolatedHTransformControlMode(pre_grasp_pose, controller_name = 'GoAboveObject', goal_is_relative='0', name = 'Pregrasp'))
- 
+    control_sequence.append(ha.InterpolatedHTransformControlMode(pre_grasp_pose,
+                                                                 controller_name = 'GoAboveObject',
+                                                                 goal_is_relative='0',
+                                                                 name = 'Pregrasp',
+                                                                 v_max = pre_grasp_velocity))
```

Comment:
why do we need to reduce the arm velocity in free space motion?

Reply:
Good you pointed that out. It was initially intended as a hotfix for #29 . A similar problem occured during pre grasp. This was added after we discussed the hope that an decrease in EE velocity will also decrease the joint space in general. It is clear that this is not necessarily the case.
However, the part that adds this parameter to PostGraspRotate is completely missing in this commit.
Should I add it as well or do we wan't to throw away both velocity max parameters?